test/functional/wallet_bumpfee.py

```diff
@@ -545,9 +545,9 @@ def test_settxfee(self, rbf_node, dest_address):
     assert_greater_than(Decimal("0.00001000"), abs(requested_feerate - actual_feerate))
     rbf_node.settxfee(Decimal("0.00000000"))  # unset paytxfee
 
-    # check that settxfee respects -maxtxfee
-    self.restart_node(1, ['-maxtxfee=0.000025'] + self.extra_args[1])
-    assert_raises_rpc_error(-8, "txfee cannot be more than wallet max tx fee", rbf_node.settxfee, Decimal('0.00003'))
```

Comment:
in "wallet: add maxfeerate wallet startup option" https://github.com/bitcoin/bitcoin/commit/4165dc41289fd337e279f3986dfc1a84345c900e

After adding `maxfeerate` option we now only check if the fee rate user set is less  `minrelaytxfee`, less than wallet min fee, or whether it exceed `maxfeerate`.
So their is no need to check for `maxtxfee` here, because its not enforced here.

Will add this info in commit message, thanks

Reply:
Added a test for this thanks.